client/cli/go/cmds/block_volume.go

```diff
@@ -250,17 +250,28 @@ var blockVolumeListCommand = &cobra.Command{
 }
 
 var blockVolumeExpandCommand = &cobra.Command{
-	Use:     "expand",
-	Short:   "Expand an existing block volume",
-	Long:    "Expand an existing block volume",
-	Example: "  $ heketi-cli blockvolume expand --block-volume=60d46d518074b13a04ce1022c8c7193c --new-size=10",
+	Use:   "expand",
+	Short: "Expand an existing block volume",
+	Long:  "Expand an existing block volume",
+	Example: `  * Expand a blockvolume to net new size 100GiB
+      $ heketi-cli blockvolume expand 60d46d518074b13a04ce1022c8c7193c --new-size=100
+                     [or, you can also use]
+      $ heketi-cli blockvolume expand --blockvolume=60d46d518074b13a04ce1022c8c7193c --new-size=100
+`,
 	RunE: func(cmd *cobra.Command, args []string) error {
+
 		if bvId == "" {
-			return errors.New("Missing --block-volume=$id")
+			s := cmd.Flags().Args()
+			if len(s) < 1 {
+				return errors.New("Missing blockvolume id")
```

Comment:
I'd put a space here to make a it read more like a sentence: "Missing block volume id"

Reply:
Sure, I will refresh.